### PR TITLE
fix: linter

### PR DESCRIPTION
**Pull request by miton18**

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-go#58.0 ✘  
* 
  * Debian:Jessie ✘  
  * Debian:Stretch ✘  
  * Debian:Wheezy ✘  
  * Ubuntu:Bionic ✘  
  * Ubuntu:Trusty ✘  
  * Ubuntu:Vivid ✘  
  * Ubuntu:Xenial ✘  
  * Ubuntu:Yakkety ✘

File: Makefile [hunk 1]
```diff
@@ -48,7 +48,8 @@ clean:
 
 build-docker: build-go-in-docker build-docker-image
 
-glide-install:
```
Comment (aurrelhebert) [inline Makefile]: Please ensure all glide-install references are discards

**Comment by ovh-cds** [comment]
CDS Report build-go#61.0 ✘  
* 
  * Debian:Jessie ✘  
  * Debian:Stretch ✘  
  * Debian:Wheezy ✘  
  * Ubuntu:Bionic ✘  
  * Ubuntu:Trusty ✘  
  * Ubuntu:Vivid ✘  
  * Ubuntu:Xenial ✘  
  * Ubuntu:Yakkety ✘

**Comment by FlorentinDUBOIS** [comment]
Resolved with PR #61

**Pull request closed without merging**